module/classifier/predict.py

```diff
@@ -90,24 +92,22 @@ def __load_model(self):
 
     def __get_prediction(
         self, embedded_question
-    ) -> Tuple[str, str, List[Media], float]:
+    ) -> Tuple[str, str, Media, Media, Media, float]:
         prediction = self.model.predict([embedded_question])
         decision = self.model.decision_function([embedded_question])
         highest_confidence = max(decision[0])
         answer_text = self.mentor.answer_id_by_answer[prediction[0]]
         answer_key = sanitize_string(answer_text)
-        answer_media = (
-            self.mentor.questions_by_answer[answer_key].get("media", [])
-            if answer_key in self.mentor.questions_by_answer
-            else []
-        )
-        return prediction[0], answer_text, answer_media, float(highest_confidence)
+        answer_web_media = self.mentor.questions_by_answer[answer_key].get("webMedia")
+        answer_mobile_media = self.mentor.questions_by_answer[answer_key].get("mobileMedia")
+        answer_vtt_media = self.mentor.questions_by_answer[answer_key].get("vttMedia")
+        return prediction[0], answer_text, answer_web_media, answer_mobile_media, answer_vtt_media, float(highest_confidence)
```

Comment:
this is probably an anti-pattern, a better approach is to create a data class or something similar to group them by name